app/resources/numismatic_finds/numismatic_find_change_set.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+class NumismaticFindChangeSet < Valkyrie::ChangeSet
+  delegate :human_readable_type, to: :model
+
+  property :place, multiple: false, required: false
+  property :date, multiple: false, required: false
+  property :find_number, multiple: false, required: false
+  property :feature, multiple: false, required: false
+  property :locus, multiple: false, required: false
+  property :description, multiple: false, required: false
+
+  validates_with AutoIncrementValidator, property: :find_number
+
+  def primary_terms
+    [
+      :place,
+      :date,
+      :find_number,
```

Comment:
I think we don't want users to be able to edit the `find_number`, so it shouldn't be in here.